web/ui/react-app/src/Checkbox.test.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Checkbox from './Checkbox';
+import { FormGroup, Label, Input } from 'reactstrap';
+
+const mockEvent: React.ChangeEvent<HTMLInputElement> = {
+  currentTarget: {},
+} as React.ChangeEvent<HTMLInputElement>;
+
+const MockCmp: React.FC = () => <div className={'mock'} />;
```

Comment:
You're right. It's better to simplify it (e.g., `className="mock"`) if the attribute value is constant. This was just a mistake, likely because I'm so used to using braces to embed expressions in jsx. I will change this and other things like it

Reply:
Another unrelated PS on `className`s. Generally I've seen that it is good practice to namespace `className`s if they are being used as css style selectors. [SuitCss](https://suitcss.github.io/) and the corresponding [react library](https://github.com/acdlite/react-suitcss) can work well for this. 

Another newer cool approach is to use [Styled-Components](https://www.styled-components.com/), which allows you to write your styles directly into your javascript. At that point, you don't need to bind styles via `className`s anymore, so they often aren't really necessary

Reply:
Right, I ran into a similar in-JS styling approach with some Material UI code recently. I've yet to develop a strong opinion on where things should live, but I'm open to improvements (in other PRs, of course) :)

Reply:
Regarding currly braces around strings, this plugin can be useful https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/jsx-curly-brace-presence.md

Reply:
@boyskila cool! We should consider adding that to the eslint config.